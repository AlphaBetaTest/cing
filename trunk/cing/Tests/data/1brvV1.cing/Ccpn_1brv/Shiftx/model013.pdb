REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       4.323   1.949   4.177  1.00  0.00              
ATOM      2  CA  VAL A 171       4.310   0.876   3.132  1.00  0.00              
ATOM      3  HA  VAL A 171       4.245   1.457   2.216  1.00  0.00              
ATOM      4  CB  VAL A 171       5.698   0.145   3.164  1.00  0.00              
ATOM      5  HB  VAL A 171       6.458   0.945   3.191  1.00  0.00              
ATOM      6  CG1 VAL A 171       5.897  -0.779   4.386  1.00  0.00              
ATOM      7 HG11 VAL A 171       5.722  -0.241   5.335  1.00  0.00              
ATOM      8 HG12 VAL A 171       5.202  -1.638   4.369  1.00  0.00              
ATOM      9 HG13 VAL A 171       6.924  -1.184   4.437  1.00  0.00              
ATOM     10  CG2 VAL A 171       6.095  -0.634   1.894  1.00  0.00              
ATOM     11 HG21 VAL A 171       5.361  -1.403   1.601  1.00  0.00              
ATOM     12 HG22 VAL A 171       6.202   0.041   1.031  1.00  0.00              
ATOM     13 HG23 VAL A 171       7.074  -1.139   1.991  1.00  0.00              
ATOM     14  C   VAL A 171       3.051  -0.062   2.939  1.00  0.00              
ATOM     15  O   VAL A 171       3.003  -0.707   1.892  1.00  0.00              
ATOM     16  N   PRO A 172       2.018  -0.225   3.806  1.00  0.00              
ATOM     17  CD  PRO A 172       2.065   0.236   5.190  1.00  0.00              
ATOM     18  CA  PRO A 172       0.796  -1.004   3.472  1.00  0.00              
ATOM     19  HA  PRO A 172       1.017  -1.846   2.785  1.00  0.00              
ATOM     20  CB  PRO A 172       0.449  -1.594   4.854  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.638  -1.744   4.995  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.908  -2.599   4.942  1.00  0.00              
ATOM     23  CG  PRO A 172       1.058  -0.663   5.915  1.00  0.00              
ATOM     24  HG2 PRO A 172       0.282  -0.055   6.409  1.00  0.00              
ATOM     25  HG3 PRO A 172       1.546  -1.241   6.722  1.00  0.00              
ATOM     26  HD2 PRO A 172       1.801   1.315   5.201  1.00  0.00              
ATOM     27  HD3 PRO A 172       3.088   0.132   5.602  1.00  0.00              
ATOM     28  C   PRO A 172      -0.291  -0.074   2.821  1.00  0.00              
ATOM     29  O   PRO A 172      -0.899   0.755   3.508  1.00  0.00              
ATOM     30  N   CYS A 173      -0.542  -0.205   1.501  1.00  0.00              
ATOM     31  H   CYS A 173       0.004  -0.910   0.997  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.539   0.620   0.748  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.203   1.657   0.756  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.425   0.251  -0.733  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.800  -0.627  -0.912  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.411   0.058  -1.132  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.613   1.539  -1.674  1.00  0.00              
ATOM     38  C   CYS A 173      -3.001   0.704   1.308  1.00  0.00              
ATOM     39  O   CYS A 173      -3.620   1.771   1.223  1.00  0.00              
ATOM     40  N   SER A 174      -3.525  -0.377   1.912  1.00  0.00              
ATOM     41  H   SER A 174      -2.888  -1.182   1.908  1.00  0.00              
ATOM     42  CA  SER A 174      -4.792  -0.329   2.717  1.00  0.00              
ATOM     43  HA  SER A 174      -5.609  -0.053   2.052  1.00  0.00              
ATOM     44  CB  SER A 174      -5.091  -1.747   3.258  1.00  0.00              
ATOM     45  HB2 SER A 174      -6.067  -1.735   3.784  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.213  -2.456   2.418  1.00  0.00              
ATOM     47  OG  SER A 174      -4.073  -2.216   4.150  1.00  0.00              
ATOM     48  HG  SER A 174      -4.419  -2.996   4.598  1.00  0.00              
ATOM     49  C   SER A 174      -4.893   0.744   3.858  1.00  0.00              
ATOM     50  O   SER A 174      -5.979   1.221   4.187  1.00  0.00              
ATOM     51  N   THR A 175      -3.743   1.119   4.415  1.00  0.00              
ATOM     52  H   THR A 175      -2.937   0.614   4.028  1.00  0.00              
ATOM     53  CA  THR A 175      -3.595   2.156   5.476  1.00  0.00              
ATOM     54  HA  THR A 175      -4.564   2.250   6.007  1.00  0.00              
ATOM     55  CB  THR A 175      -2.554   1.659   6.526  1.00  0.00              
ATOM     56  HB  THR A 175      -2.643   2.311   7.419  1.00  0.00              
ATOM     57  OG1 THR A 175      -1.223   1.780   6.031  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.206   1.368   5.155  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.744   0.196   6.978  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.787   0.012   7.296  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.559  -0.507   6.139  1.00  0.00              
ATOM     62 HG23 THR A 175      -2.063  -0.083   7.793  1.00  0.00              
ATOM     63  C   THR A 175      -3.267   3.593   4.929  1.00  0.00              
ATOM     64  O   THR A 175      -3.574   4.583   5.596  1.00  0.00              
ATOM     65  N   CYS A 176      -2.629   3.723   3.740  1.00  0.00              
ATOM     66  H   CYS A 176      -2.660   2.880   3.161  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.444   5.016   3.043  1.00  0.00              
ATOM     68  HA  CYS A 176      -1.980   5.747   3.735  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.447   4.677   1.919  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.822   3.970   1.189  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.243   5.536   1.267  1.00  0.00              
ATOM     72  SG  CYS A 176       0.115   4.092   2.605  1.00  0.00              
ATOM     73  C   CYS A 176      -3.806   5.628   2.563  1.00  0.00              
ATOM     74  O   CYS A 176      -4.030   6.830   2.710  1.00  0.00              
ATOM     75  N   GLU A 177      -4.695   4.788   1.989  1.00  0.00              
ATOM     76  H   GLU A 177      -4.281   3.905   1.680  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.123   5.093   1.696  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.394   4.329   0.951  1.00  0.00              
ATOM     79  CB  GLU A 177      -6.958   4.766   2.967  1.00  0.00              
ATOM     80  HB2 GLU A 177      -8.020   4.780   2.666  1.00  0.00              
ATOM     81  HB3 GLU A 177      -6.740   3.721   3.256  1.00  0.00              
ATOM     82  CG  GLU A 177      -6.787   5.674   4.217  1.00  0.00              
ATOM     83  HG2 GLU A 177      -5.739   5.659   4.562  1.00  0.00              
ATOM     84  HG3 GLU A 177      -6.989   6.729   3.957  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.642   5.266   5.412  1.00  0.00              
ATOM     86  OE1 GLU A 177      -7.458   4.252   6.077  1.00  0.00              
ATOM     87  OE2 GLU A 177      -8.633   6.161   5.672  1.00  0.00              
ATOM     88  C   GLU A 177      -6.506   6.424   0.992  1.00  0.00              
ATOM     89  O   GLU A 177      -7.475   7.112   1.325  1.00  0.00              
ATOM     90  N   GLY A 178      -5.745   6.710  -0.058  1.00  0.00              
ATOM     91  H   GLY A 178      -4.839   6.235   0.015  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.868   7.973  -0.807  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.914   7.686  -1.867  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.818   8.472  -0.586  1.00  0.00              
ATOM     95  C   GLY A 178      -4.730   9.005  -0.645  1.00  0.00              
ATOM     96  O   GLY A 178      -4.736  10.013  -1.353  1.00  0.00              
ATOM     97  N   ASN A 179      -3.744   8.768   0.242  1.00  0.00              
ATOM     98  H   ASN A 179      -3.714   7.790   0.566  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.557   9.637   0.366  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.796  10.680   0.076  1.00  0.00              
ATOM    101  CB  ASN A 179      -1.931   9.488   1.766  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.689   9.309   2.555  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.302   8.576   1.703  1.00  0.00              
ATOM    104  CG  ASN A 179      -1.003  10.628   2.149  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.160  10.673   1.767  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.489  11.591   2.879  1.00  0.00              
ATOM    107 HD21 ASN A 179      -0.809  12.314   3.121  1.00  0.00              
ATOM    108 HD22 ASN A 179      -2.465  11.491   3.173  1.00  0.00              
ATOM    109  C   ASN A 179      -1.561   9.131  -0.665  1.00  0.00              
ATOM    110  O   ASN A 179      -0.903   8.097  -0.502  1.00  0.00              
ATOM    111  N   LEU A 180      -1.445   9.884  -1.736  1.00  0.00              
ATOM    112  H   LEU A 180      -2.089  10.676  -1.808  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.652   9.428  -2.870  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.070   8.415  -2.964  1.00  0.00              
ATOM    115  CB  LEU A 180      -1.058  10.384  -4.002  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.236  11.403  -3.585  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.205  10.459  -4.646  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.228   9.963  -4.897  1.00  0.00              
ATOM    119  HG  LEU A 180      -3.041   9.801  -4.192  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.613  11.094  -5.864  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.863  12.030  -5.329  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.796  11.331  -6.573  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.501  10.831  -6.469  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.964   8.677  -5.702  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.757   7.807  -5.053  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.834   8.398  -6.326  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.098   8.784  -6.383  1.00  0.00              
ATOM    128  C   LEU A 180       0.881   9.209  -2.684  1.00  0.00              
ATOM    129  O   LEU A 180       1.485   8.408  -3.389  1.00  0.00              
ATOM    130  N   ALA A 181       1.460   9.895  -1.704  1.00  0.00              
ATOM    131  H   ALA A 181       0.735  10.410  -1.207  1.00  0.00              
ATOM    132  CA  ALA A 181       2.828   9.631  -1.214  1.00  0.00              
ATOM    133  HA  ALA A 181       3.486   9.556  -2.091  1.00  0.00              
ATOM    134  CB  ALA A 181       3.249  10.860  -0.408  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.206  11.776  -1.023  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.602  11.003   0.478  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.287  10.741  -0.052  1.00  0.00              
ATOM    138  C   ALA A 181       2.934   8.288  -0.408  1.00  0.00              
ATOM    139  O   ALA A 181       3.778   7.458  -0.738  1.00  0.00              
ATOM    140  N   CYS A 182       2.062   8.051   0.598  1.00  0.00              
ATOM    141  H   CYS A 182       1.243   8.669   0.556  1.00  0.00              
ATOM    142  CA  CYS A 182       1.945   6.739   1.312  1.00  0.00              
ATOM    143  HA  CYS A 182       2.814   6.489   1.881  1.00  0.00              
ATOM    144  CB  CYS A 182       0.908   6.985   2.403  1.00  0.00              
ATOM    145  HB2 CYS A 182       1.063   7.946   2.903  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.081   7.059   1.958  1.00  0.00              
ATOM    147  SG  CYS A 182       0.859   5.643   3.600  1.00  0.00              
ATOM    148  C   CYS A 182       1.707   5.513   0.375  1.00  0.00              
ATOM    149  O   CYS A 182       2.403   4.496   0.425  1.00  0.00              
ATOM    150  N   LEU A 183       0.731   5.682  -0.508  1.00  0.00              
ATOM    151  H   LEU A 183       0.209   6.562  -0.398  1.00  0.00              
ATOM    152  CA  LEU A 183       0.423   4.768  -1.617  1.00  0.00              
ATOM    153  HA  LEU A 183       0.155   3.787  -1.195  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.794   5.473  -2.253  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.515   6.507  -2.505  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.963   5.053  -3.203  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.175   5.431  -1.596  1.00  0.00              
ATOM    158  HG  LEU A 183      -2.018   5.923  -0.622  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.161   6.213  -2.480  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.133   5.873  -3.534  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.204   6.093  -2.151  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.941   7.295  -2.488  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.724   4.003  -1.445  1.00  0.00              
ATOM    164 HD21 LEU A 183      -2.819   3.487  -2.420  1.00  0.00              
ATOM    165 HD22 LEU A 183      -2.068   3.373  -0.827  1.00  0.00              
ATOM    166 HD23 LEU A 183      -3.716   3.984  -0.960  1.00  0.00              
ATOM    167  C   LEU A 183       1.558   4.545  -2.684  1.00  0.00              
ATOM    168  O   LEU A 183       1.667   3.455  -3.245  1.00  0.00              
ATOM    169  N   SER A 184       2.368   5.574  -2.984  1.00  0.00              
ATOM    170  H   SER A 184       2.158   6.393  -2.394  1.00  0.00              
ATOM    171  CA  SER A 184       3.573   5.437  -3.866  1.00  0.00              
ATOM    172  HA  SER A 184       3.270   4.885  -4.778  1.00  0.00              
ATOM    173  CB  SER A 184       4.035   6.835  -4.331  1.00  0.00              
ATOM    174  HB2 SER A 184       3.202   7.371  -4.824  1.00  0.00              
ATOM    175  HB3 SER A 184       4.329   7.461  -3.465  1.00  0.00              
ATOM    176  OG  SER A 184       5.121   6.741  -5.258  1.00  0.00              
ATOM    177  HG  SER A 184       5.670   5.996  -4.980  1.00  0.00              
ATOM    178  C   SER A 184       4.760   4.612  -3.251  1.00  0.00              
ATOM    179  O   SER A 184       5.538   4.023  -4.005  1.00  0.00              
ATOM    180  N   LEU A 185       4.932   4.604  -1.918  1.00  0.00              
ATOM    181  H   LEU A 185       4.262   5.198  -1.403  1.00  0.00              
ATOM    182  CA  LEU A 185       5.916   3.707  -1.227  1.00  0.00              
ATOM    183  HA  LEU A 185       6.846   3.628  -1.794  1.00  0.00              
ATOM    184  CB  LEU A 185       6.220   4.245   0.208  1.00  0.00              
ATOM    185  HB2 LEU A 185       5.346   4.015   0.851  1.00  0.00              
ATOM    186  HB3 LEU A 185       7.047   3.649   0.642  1.00  0.00              
ATOM    187  CG  LEU A 185       6.536   5.745   0.393  1.00  0.00              
ATOM    188  HG  LEU A 185       5.644   6.281   0.015  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.706   6.087   1.881  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.803   5.829   2.465  1.00  0.00              
ATOM    191 HD12 LEU A 185       7.559   5.550   2.338  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.880   7.169   2.032  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.776   6.199  -0.392  1.00  0.00              
ATOM    194 HD21 LEU A 185       7.656   6.024  -1.478  1.00  0.00              
ATOM    195 HD22 LEU A 185       7.964   7.281  -0.270  1.00  0.00              
ATOM    196 HD23 LEU A 185       8.692   5.665  -0.076  1.00  0.00              
ATOM    197  C   LEU A 185       5.509   2.212  -1.096  1.00  0.00              
ATOM    198  O   LEU A 185       6.376   1.344  -0.977  1.00  0.00              
ATOM    199  N   CYS A 186       4.204   1.931  -1.091  1.00  0.00              
ATOM    200  H   CYS A 186       3.596   2.746  -1.064  1.00  0.00              
ATOM    201  CA  CYS A 186       3.662   0.637  -0.740  1.00  0.00              
ATOM    202  HA  CYS A 186       4.212   0.423   0.190  1.00  0.00              
ATOM    203  CB  CYS A 186       2.170   1.003  -0.487  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.719   0.236   0.145  1.00  0.00              
ATOM    205  HB3 CYS A 186       1.969   1.926   0.072  1.00  0.00              
ATOM    206  SG  CYS A 186       1.253   0.978  -2.035  1.00  0.00              
ATOM    207  C   CYS A 186       3.770  -0.593  -1.700  1.00  0.00              
ATOM    208  O   CYS A 186       4.288  -0.564  -2.819  1.00  0.00              
ATOM    209  N   HIS A 187       3.148  -1.676  -1.220  1.00  0.00              
ATOM    210  H   HIS A 187       2.992  -1.629  -0.205  1.00  0.00              
ATOM    211  CA  HIS A 187       2.920  -2.921  -1.985  1.00  0.00              
ATOM    212  HA  HIS A 187       3.654  -2.999  -2.813  1.00  0.00              
ATOM    213  CB  HIS A 187       3.026  -4.215  -1.117  1.00  0.00              
ATOM    214  HB2 HIS A 187       2.050  -4.691  -0.980  1.00  0.00              
ATOM    215  HB3 HIS A 187       3.584  -4.924  -1.737  1.00  0.00              
ATOM    216  CG  HIS A 187       3.628  -4.190   0.276  1.00  0.00              
ATOM    217  ND1 HIS A 187       2.910  -3.740   1.367  1.00  0.00              
ATOM    218  CD2 HIS A 187       4.945  -4.470   0.618  1.00  0.00              
ATOM    219  CE1 HIS A 187       3.911  -3.723   2.288  1.00  0.00              
ATOM    220  NE2 HIS A 187       5.159  -4.187   1.957  1.00  0.00              
ATOM    221  HD2 HIS A 187       5.668  -4.789  -0.112  1.00  0.00              
ATOM    222  HE1 HIS A 187       3.754  -3.108   3.160  1.00  0.00              
ATOM    223  C   HIS A 187       1.459  -2.826  -2.560  1.00  0.00              
ATOM    224  O   HIS A 187       0.482  -2.739  -1.806  1.00  0.00              
ATOM    225  N   ILE A 188       1.304  -2.879  -3.888  1.00  0.00              
ATOM    226  H   ILE A 188       2.196  -2.843  -4.392  1.00  0.00              
ATOM    227  CA  ILE A 188      -0.044  -2.774  -4.548  1.00  0.00              
ATOM    228  HA  ILE A 188      -0.765  -2.317  -3.843  1.00  0.00              
ATOM    229  CB  ILE A 188      -0.029  -1.768  -5.760  1.00  0.00              
ATOM    230  HB  ILE A 188       0.751  -2.118  -6.453  1.00  0.00              
ATOM    231  CG2 ILE A 188      -1.368  -1.761  -6.550  1.00  0.00              
ATOM    232 HG21 ILE A 188      -1.621  -2.756  -6.962  1.00  0.00              
ATOM    233 HG22 ILE A 188      -2.221  -1.454  -5.914  1.00  0.00              
ATOM    234 HG23 ILE A 188      -1.345  -1.076  -7.418  1.00  0.00              
ATOM    235  CG1 ILE A 188       0.263  -0.277  -5.393  1.00  0.00              
ATOM    236 HG12 ILE A 188      -0.111   0.403  -6.185  1.00  0.00              
ATOM    237 HG13 ILE A 188      -0.316   0.011  -4.494  1.00  0.00              
ATOM    238  CD1 ILE A 188       1.742   0.098  -5.195  1.00  0.00              
ATOM    239 HD11 ILE A 188       2.367  -0.167  -6.068  1.00  0.00              
ATOM    240 HD12 ILE A 188       1.861   1.187  -5.035  1.00  0.00              
ATOM    241 HD13 ILE A 188       2.189  -0.385  -4.310  1.00  0.00              
ATOM    242  C   ILE A 188      -0.620  -4.204  -4.876  1.00  0.00              
ATOM    243  O   ILE A 188      -1.756  -4.495  -4.497  1.00  0.00              
ATOM    244  N   GLU A 189       0.117  -5.084  -5.584  1.00  0.00              
ATOM    245  H   GLU A 189       1.014  -4.728  -5.907  1.00  0.00              
ATOM    246  CA  GLU A 189      -0.352  -6.449  -5.942  1.00  0.00              
ATOM    247  HA  GLU A 189      -1.352  -6.348  -6.398  1.00  0.00              
ATOM    248  CB  GLU A 189       0.598  -7.054  -7.013  1.00  0.00              
ATOM    249  HB2 GLU A 189       1.630  -7.101  -6.607  1.00  0.00              
ATOM    250  HB3 GLU A 189       0.317  -8.116  -7.169  1.00  0.00              
ATOM    251  CG  GLU A 189       0.608  -6.360  -8.401  1.00  0.00              
ATOM    252  HG2 GLU A 189      -0.414  -6.331  -8.821  1.00  0.00              
ATOM    253  HG3 GLU A 189       0.919  -5.303  -8.311  1.00  0.00              
ATOM    254  CD  GLU A 189       1.512  -7.049  -9.417  1.00  0.00              
ATOM    255  OE1 GLU A 189       1.129  -7.911 -10.200  1.00  0.00              
ATOM    256  OE2 GLU A 189       2.796  -6.606  -9.355  1.00  0.00              
ATOM    257  C   GLU A 189      -0.476  -7.403  -4.713  1.00  0.00              
TER     258      GLU A 189 
END   
