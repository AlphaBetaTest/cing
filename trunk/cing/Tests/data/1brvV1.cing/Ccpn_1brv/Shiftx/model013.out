# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      120.4472
  171   V  CA      60.6219
  171   V  CB      34.6928
  171   V  C      174.5912
  171   V  H        7.8976
  171   V  HA       4.1112
  171   V  HB       1.9475
  171   V  HG11     0.9446
  171   V  HG12     0.9231
  171   V  HG13     0.4250
  171   V  HG21     0.5019
  171   V  HG22     1.0374
  171   V  HG23     0.8315
  172   P  CA      61.1783
  172   P  CB      31.3953
  172   P  C      175.5288
  172   P  HA       4.6662
  172   P  HB2      2.3427
  172   P  HB3      2.3024
  172   P  HD2      3.7189
  172   P  HD3      3.7812
  172   P  HG2      2.0436
  172   P  HG3      2.0710
  173   C  N      122.9762
  173   C  CA      60.1385
  173   C  CB      40.7607
  173   C  C      177.2767
  173   C  H        8.8442
  173   C  HA       4.2054
  173   C  HB2      3.1734
  173   C  HB3      3.2050
  174   S  N      114.6771
  174   S  CA      61.4462
  174   S  CB      62.2675
  174   S  C      175.8951
  174   S  H        8.2290
  174   S  HA       3.9791
  174   S  HB2      4.0205
  174   S  HB3      4.0301
  175   T  N      111.6612
  175   T  CA      63.5199
  175   T  CB      69.5296
  175   T  C      176.1562
  175   T  H        7.7530
  175   T  HA       4.2783
  175   T  HB       4.2444
  175   T  HG21     1.2158
  175   T  HG22     1.2200
  175   T  HG23     1.2175
  176   C  N      119.5078
  176   C  CA      58.4408
  176   C  CB      40.6449
  176   C  C      174.0624
  176   C  H        8.1895
  176   C  HA       4.3700
  176   C  HB2      3.1315
  176   C  HB3      3.2176
  177   E  N      115.8794
  177   E  CA      57.6238
  177   E  CB      28.4551
  177   E  C      175.3468
  177   E  H        9.1019
  177   E  HA       3.6482
  177   E  HB2      2.1734
  177   E  HB3      2.2075
  177   E  HG2      2.3838
  177   E  HG3      2.2326
  178   G  N      104.4365
  178   G  CA      46.3745
  178   G  C      174.0657
  178   G  H        8.3790
  178   G  HA    -666.0000
  178   G  HA2      3.9600
  178   G  HA3      3.9599
  179   N  N      118.9561
  179   N  CA      52.5171
  179   N  CB      39.2148
  179   N  C      175.2110
  179   N  H        8.1963
  179   N  HA       4.5828
  179   N  HB2      2.8894
  179   N  HB3      2.7334
  179   N  HD21     7.1030
  179   N  HD22     6.5682
  180   L  N      129.1596
  180   L  CA      57.2551
  180   L  CB      41.6177
  180   L  C      179.4386
  180   L  H        8.4557
  180   L  HA       3.9674
  180   L  HB2      1.7241
  180   L  HB3      1.7023
  180   L  HD11     0.9189
  180   L  HD12     0.9188
  180   L  HD13     0.9191
  180   L  HD21     0.2276
  180   L  HD22     1.8522
  180   L  HD23     0.8683
  180   L  HG       1.7048
  181   A  N      120.7351
  181   A  CA      55.1999
  181   A  CB      18.2437
  181   A  C      179.6832
  181   A  H        7.9023
  181   A  HA       3.9018
  181   A  HB1      1.3858
  181   A  HB2      1.3530
  181   A  HB3      1.3312
  182   C  N      116.8554
  182   C  CA      60.1298
  182   C  CB      43.4669
  182   C  C      175.9071
  182   C  H        7.9932
  182   C  HA       4.1243
  182   C  HB2      2.9829
  182   C  HB3      3.2315
  183   L  N      119.0969
  183   L  CA      57.8703
  183   L  CB      41.2193
  183   L  C      179.2562
  183   L  H        7.9374
  183   L  HA       3.7191
  183   L  HB2      1.8633
  183   L  HB3      1.7161
  183   L  HD11     0.9190
  183   L  HD12     0.9960
  183   L  HD13     0.9035
  183   L  HD21     0.3498
  183   L  HD22     2.3494
  183   L  HD23     0.8707
  183   L  HG       1.6807
  184   S  N      112.3266
  184   S  CA      61.0909
  184   S  CB      62.3652
  184   S  C      175.8347
  184   S  H        7.9021
  184   S  HA       4.2279
  184   S  HB2      4.0024
  184   S  HB3      4.0152
  185   L  N      121.4221
  185   L  CA      57.0232
  185   L  CB      41.6625
  185   L  C      174.7998
  185   L  H        7.5776
  185   L  HA       4.1794
  185   L  HB2      1.5771
  185   L  HB3      1.6835
  185   L  HD11     0.9078
  185   L  HD12     0.9081
  185   L  HD13     0.9113
  185   L  HD21     0.3122
  185   L  HD22     1.5533
  185   L  HD23     0.8586
  185   L  HG       1.7993
  186   C  N      112.0505
  186   C  CA      54.5079
  186   C  CB      42.9898
  186   C  C      171.8678
  186   C  H        7.3773
  186   C  HA       5.1599
  186   C  HB2      3.3099
  186   C  HB3      3.2903
  187   H  N      116.7937
  187   H  CA      54.9569
  187   H  CB      29.2304
  187   H  C      174.9037
  187   H  H        8.2245
  187   H  HA       4.8116
  187   H  HB2      3.1258
  187   H  HB3      3.2938
  187   H  HD2      5.8804
  187   H  HE1      6.8011
  188   I  N      127.7184
  188   I  CA      63.7512
  188   I  CB      38.0918
  188   I  C      176.3220
  188   I  H        8.5605
  188   I  HA       3.9518
  188   I  HB       1.8771
  188   I  HD11     0.8597
  188   I  HD12     0.8545
  188   I  HD13     0.8563
  188   I  HG12     1.2749
  188   I  HG13     0.4013
  188   I  HG21     0.9163
  188   I  HG22     0.9158
  188   I  HG23     0.9139
 *189   E  N      118.7103
 *189   E  CA      56.0929
 *189   E  CB      30.2529
 *189   E  C      177.2668
 *189   E  H        8.0051
 *189   E  HA       4.1586
 *189   E  HB2      1.7414
 *189   E  HB3      1.7683
 *189   E  HG2      2.1327
 *189   E  HG3      2.3809
